#!/usr/bin/env python
PACKAGE = "velodyne_pointcloud"

from math import pi
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_range", double_t, 0, "min range to publish", 0.9, 0.1, 10.0)
gen.add("max_range", double_t, 0, "max range to publish", 130, 0.1, 200)
gen.add("view_direction", double_t, 0, "angle defining the center of view",
        0.0, -pi, pi)
gen.add("view_width", double_t, 0, "angle defining the view width",
        2*pi, 0.0, 2*pi)

exit(gen.generate(PACKAGE, "cloud_node", "CloudNode"))
